lusidtools/apps/flush_transactions.py

```diff
@@ -186,6 +186,45 @@ def get_portfolios_from_group(expanded_group):
     return temp_portfolio_lst
 
 
+def get_transactions_from_portfolio_list(args, portfolio_list, transaction_portfolios_api):
```

Comment:
I think its worth renaming this to `get_all_transactions_from_portfolio_list` and adding a comment to the docstring noting that is resolves pagination